### PR TITLE
add teammate tracking actionbar toggle option

### PR DESCRIPTION
**Pull request by ImYenil**
added teammate tracking actionbar toggle option

### CODE REVIEW

File: bedwars-plugin/src/main/java/com/tomkeuper/bedwars/arena/tasks/GamePlayingTask.java [hunk 1]
```diff
@@ -175,36 +176,37 @@ public void run() {
...
+            for (Player p : getArena().getPlayers()) {
+                if (Arena.afkCheck.get(p.getUniqueId()) == null) {
+                    Arena.afkCheck.put(p.getUniqueId(), current);
+                } else {
+                    current = Arena.afkCheck.get(p.getUniqueId());
+                    current++;
+                    Arena.afkCheck.replace(p.getUniqueId(), current);
+                    if (current == 45) {
+                        BedWars.getAPI().getAFKUtil().setPlayerAFK(p, true);
+                    }
```
Comment (tomkeuper) [inline bedwars-plugin/src/main/java/com/tomkeuper/bedwars/arena/tasks/GamePlayingTask.java]: It looks like you just put the afk system inside the for loop

Comment (ImYenil) [reply bedwars-plugin/src/main/java/com/tomkeuper/bedwars/arena/tasks/GamePlayingTask.java]: > It looks like you just put the afk system inside the for loop

Fixed

**Comment by tomkeuper** [comment]
btw could you add defauly off? Its not normal to track your own players in bedwars.
idk since when this showed up in builds tho

**Comment by ImYenil** [comment]
> btw could you add defauly off? Its not normal to track your own players in bedwars. idk since when this showed up in builds tho

Do you mean to set the default value to false?

**Comment by tomkeuper** [comment]
> > btw could you add defauly off? Its not normal to track your own players in bedwars. idk since when this showed up in builds tho
> 
> Do you mean to set the default value to false?

Yep

**Comment by ImYenil** [comment]
> > > btw could you add defauly off? Its not normal to track your own players in bedwars. idk since when this showed up in builds tho
> > 
> > 
> > Do you mean to set the default value to false?
> 
> Yep

done

**Review by tomkeuper — APPROVED** [review]

**Review by KKiiya — APPROVED** [review]

**Pull request merged**